.github/workflows/release.yml

```diff
@@ -128,8 +123,6 @@ jobs:
         if: runner.os == 'Linux' || runner.os == 'macOS'
         run: |
           mkdir archive
-          cp target/release/dora-coordinator archive
-          cp target/release/dora-daemon archive
           cp target/release/dora archive/dora
           cd archive
           zip -r ../archive.zip .
```

Comment:
As we only distribute a single binary as artifact, we don't need to zip it. Or is there any advantage in zipping it?

Reply:
On second thought, it seems that the best practice is to compress a binary program as it keeps the names and permissions. It also doesn't introduce a breaking change as well as let us add more file in the future if ever need be.

I think it´s best if we keep it as is.

Open for discussion.

Reply:
Ah yeah, good point about the permissions. I'm fine with keeping it in an archive.